.env.example

```diff
@@ -42,6 +50,10 @@ DISCORD_AUDIT_LOG_WEBHOOK=
 # for debugging & development purposes.
 AUTOMATICALLY_REPORT_PROBLEMS=False
 
+# XXX: change these to their location on your host server:
+SSL_CERT_PATH=/etc/ssl/certs/fullchain.pem
```

Comment:
this as a config option is kinda weird since most people use cf for ssl and so this isnt required

Reply:
yeah i have an issue open for that https://github.com/osuAkatsuki/bancho.py/issues/264

for now this is ok